Teradata/vantage-ui-platform
# fix(urlcolor): fix markdown navigator url color in dark mode

## Description
This PR takes care of url color which are displayed in the markdown navigator, for dark mode.

### What's included?
- Markdown navigator's urls are visible in dark mode.
- Markdown Navigator example under Covalent components

###Test Steps
 - Open localhost
 - Switch on the dark mode
 - Navigate to covalent components.
 - Click on `Open Markdown Navigator` button.
 - Urls inside markdown navigator should be visible.

![image](https://user-images.githubusercontent.com/12620590/84304705-c1a59400-ab0d-11ea-89bb-093a87ae9cb5.png)
